### PR TITLE
chore: add cli to readme

### PR DESCRIPTION
**Pull request by polaroi8d**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,13 +74,31 @@ Both Docker and Kubernetes require specialized staff to manage. Via dyrector.io,
...
+
+dyrectorio CLI, is a command-line interface that lets you run and manage the whole dyrectorio projects locally.
+
+This means you get a full-featured dyrectorio platform running locally on your machine.
 
 ### Prerequirements
 
--   Install Docker and Docker Compose to your machine
+- Docker installed on your system but Podman works, too. In case of Linux, you need Docker Compose, as well.
+- Go Compiler to run the CLI by its source code.
```
Comment (m8vago) [inline README.md:86]: it's

File: README.md [hunk 2]
```diff
@@ -72,13 +74,31 @@ Both Docker and Kubernetes require specialized staff to manage. Via dyrector.io,
...
+
+> **Note**
+> The CLI is only available from the codebase at the moment. Package manager installation is in the works.
 
-### Development (Local)
+1. Clone the repository to your local workdir with `git clone`
+2. Open the `/cli` folder and execute `go run . up`
+3. Open the `localhost:3000` and use the platform
+
+dyrectorios's command-line interface (CLI) lets you run a complete dyrectorio development environment locally with the following services: UI Service (crux-ui), Backend Service (crux), PostgreSQL database, Authentication, Migrations, and SMTP mail server.
```
Comment (m8vago) [inline README.md:97]: typeo: dyrectorio's command-line in....

File: README.md (hunk 1 above)
Comment (robot9706) [reply README.md]: "Go Compiler to run the CLI by _it is_ source code." sounds wrong to me 😅

Comment (polaroi8d) [reply README.md]: @m8vago refer to the apostrophe

Comment (robot9706) [reply README.md]: Yes I know, but this way the sentence means what I've quoted, so I think the original one was the correct one.

Comment (polaroi8d) [reply README.md]: @m8vago ?

Comment (m8vago) [reply README.md]: @robot9706 is right. sorry. He is the true master of "angology"

**Review by m8vago — APPROVED** [review]

**Review by robot9706 — APPROVED** [review]
😎

**Pull request merged**